node/coinstacks/solana/api/src/app.ts

```diff
@@ -1,17 +1,34 @@
 import express from 'express'
 import { join } from 'path'
 import swaggerUi from 'swagger-ui-express'
-import { middleware, Prometheus } from '@shapeshiftoss/common-api'
+import { ConnectionHandler, middleware, Prometheus, Registry, TransactionHandler } from '@shapeshiftoss/common-api'
 import { Logger } from '@shapeshiftoss/logger'
 import { RegisterRoutes } from './routes'
+import { Server } from 'ws'
+import { WebsocketClient } from './websocket'
+import { Helius } from 'helius-sdk'
+import { getTransaction } from './utils'
+import { Tx } from './models'
+import { Logs } from '@solana/web3.js'
+// import { getTransaction } from './utils'
+// import { Tx } from './models'
```

Comment:
```suggestion
```

Reply:
Fixed in [33a4238](https://github.com/shapeshift/unchained/pull/1053/commits/33a4238603fb52d1d0c78ab7639bde23265bee5f)